src/main/java/decodex/recipes/RecipeManager.java

```diff
@@ -0,0 +1,73 @@
+package decodex.recipes;
+
+import decodex.Decodex;
+import decodex.data.exception.RecipeManagerException;
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+/**
+ * RecipeManager manages a list of all recipes that were created/loaded.
+ */
+public class RecipeManager {
+
+    private Logger logger = Decodex.logger;
+
+    ArrayList<Recipe> recipeList;
+
+    /**
+     * Creates a new RecipeManager with no recipes.
+     */
+    public RecipeManager() {
+        this.recipeList = new ArrayList<>();
+    }
+
+    /**
+     * Creates a new RecipeManager with a predefined list of recipes.
+     *
+     * @param recipeList list of recipes to be loaded.
+     */
+    public RecipeManager(ArrayList<Recipe> recipeList) {
+        this.recipeList = recipeList;
+    }
+
+    /**
+     * Adds a recipe to recipe manager to be managed.
+     *
+     * @param recipe Recipe to be added to the recipe manager.
+     */
+    public void addRecipe(Recipe recipe) {
```

Comment:
The current implementation appears to allow creating recipes with duplicate names. Additional checks may be required to prevent that.